Tests/LibAudio/TestPlaybackStream.cpp

```diff
@@ -8,17 +8,31 @@
 #include <AK/MemoryStream.h>
 #include <AK/WeakPtr.h>
 #include <LibAudio/PlaybackStream.h>
+#include <LibCore/EventLoop.h>
 #include <LibTest/TestSuite.h>
 #include <unistd.h>
 
+#define STREAM_TEST TEST_CASE
+
 #if defined(HAVE_PULSEAUDIO)
 #    include <LibAudio/PulseAudioWrappers.h>
 #endif
 
-TEST_CASE(create_and_destroy_playback_stream)
+// FIXME: CI doesn't run an AudioServer currently. Creating one in /etc/SystemServer.ini does not
+//        allow this test to pass since CI runs in a Shell that will setsid() if it finds that the
+//        current session ID is 0, and AudioServer's socket address depends on the current sid.
+//        If we can fix that, this test can run on CI.
+#if defined(AK_OS_SERENITY)
+#    undef STREAM_TEST
+#    define STREAM_TEST BENCHMARK_CASE
+#endif
```

Comment:
Don't have time for a full review, so just a flyby comment - instead of defining `STREAM_TEST` above and redefining it here, you can:
```c++
#if defined(AK_OS_SERENITY)
#    define STREAM_TEST BENCHMARK_CASE
#else
#    define STREAM_TEST TEST_CASE
#endif
```